casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala

```diff
@@ -1,19 +1,33 @@
 package coop.rchain.casper
 
+import java.nio.file.{Path, Paths}
+
+import cats.effect.Sync
 import cats.implicits._
+import coop.rchain.blockstorage.util.io.IOError.RaiseIOError
+import coop.rchain.blockstorage.util.io.SourceIO
 import coop.rchain.casper.MultiParentCasper.ignoreDoppelgangerCheck
+import coop.rchain.casper.genesis.Genesis.{fromLine, getVaults}
+import coop.rchain.casper.genesis.contracts.{RevGenerator, Vault}
+import coop.rchain.casper.genesis.contracts.StandardDeploys.toDeploy
 import coop.rchain.casper.helper.HashSetCasperTestNode
 import coop.rchain.casper.helper.HashSetCasperTestNode._
 import coop.rchain.casper.scalatestcontrib._
 import coop.rchain.casper.util.ConstructDeploy
+import coop.rchain.casper.util.ConstructDeploy.defaultPub
 import coop.rchain.casper.util.GenesisBuilder.buildGenesis
 import coop.rchain.casper.util.RSpaceUtil._
 import coop.rchain.casper.util.rholang.RegistrySigGen
 import coop.rchain.crypto.PublicKey
 import coop.rchain.crypto.codec.Base16
+import coop.rchain.rholang.interpreter.util.RevAddress
+import coop.rchain.shared.Log
 import monix.execution.Scheduler.Implicits.global
 import org.scalatest.{FlatSpec, Matchers}
 
+import scala.io.Source
+import scala.util.{Failure, Success, Try}
+
 class RholangBuildTest extends FlatSpec with Matchers {
 
   val genesis = buildGenesis()
```

Comment:
Here you're constructing the entire genesis block.

Reply:
It is created with I think 4 deploys and takes only a few seconds.

Reply:
yeah - the wallets should be passed in here as parameter to genesisParameters, that are passed to buildGenesis

Reply:
(see the default parameters - we need to pass them explicitly)